.github/scripts/build_info.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
```

Comment:
nit: May be useful to have a more specific script name than `build_info` given that the name sounds generic compared to the very specific task this script has

Reply:
The developer will be able to force dependency updates for their PR using the upgrade dependencies label. I want to develop this script and add logic that sets additional variables, not just python versions in separate PRs.

Specifically, I want to implement the following requirements from the spec.
> The developer will be able to force dependency updates for their PR using the `upgrade dependencies` label.